static/src/js/components/Timeline/__tests__/Timeline.test.jsx

```diff
@@ -62,19 +67,19 @@ afterEach(() => {
 
 describe('Timeline component', () => {
   test('should render an EDSCTimeline component with the correct props', () => {
-    const { enzymeWrapper } = setup()
-    const timeline = enzymeWrapper.find(EDSCTimeline)
-
-    expect(timeline.props().center).toEqual(1609495200000)
-    expect(timeline.props().minZoom).toEqual(1)
-    expect(timeline.props().maxZoom).toEqual(5)
-    expect(timeline.props().zoom).toEqual(3)
-    expect(timeline.props().data).toEqual([])
-    expect(timeline.props().focusedInterval).toEqual({})
-    expect(timeline.props().temporalRange).toEqual({})
+    setup()
+    const timelineProps = EDSCTimeline.mock.calls[0][0]
```

Comment:
change this expect(EDSCTimeline).toHaveBeenCalledWith(expect.objectContaining(...